src/resnext.jl

```diff
@@ -0,0 +1,117 @@
+"""
+    resnextblock(inplanes, outplanes, cardinality, downsample = false)
+
+Create a basic residual block as defined in the paper for ResNeXt
+([reference](https://arxiv.org/abs/1611.05431)).
+
+# Arguments:
+- `inplanes`: the number of input feature maps
+- `outplanes`: the number of output feature maps 
+- `cardinality`: the number of groups to use for the convolution
+- `width`: the number of feature maps in each group in the bottleneck
+- `downsample`: set to `true` to downsample the input
+"""
+function resnextblock(inplanes, outplanes, cardinality, width, widen_factor, downsample = false)
+    stride = downsample ? 2 : 1
+
+    width_ratio = outplanes / (widen_factor * 128.0)
+    hidden_channels = cardinality * floor(width * width_ratio)
+
+    return Chain(conv_bn((1, 1), inplanes, hidden_channels; stride = 1, bias = false)...,
+        conv_bn((3, 3), hidden_channels, hidden_channels;
+            stride = stride, pad = 1, bias = false, groups = cardinality)...,
+        conv_bn((1, 1), hidden_channels, outplanes; stride = 1, bias = false)...)
+end
+
+"""
+    resnext(connection = (x, y) -> @. relu(x) + relu(y); channel_config, block_config, 
+            cardinality, nclasses = 1000)
+    
+Create a ResNeXt model
+([reference](https://arxiv.org/abs/1611.05431)).
+
+# Arguments
+- `cardinality`: the number of groups to use for the convolution
+- `width`: the number of feature maps in each group in the bottleneck
+- `connection`: the binary function applied to the output of residual and skip paths in a block
+- `block_config`: a list of the number of residual blocks at each stage
+- `nclasses`: the number of output classes
+"""
+function resnext(cardinality, width, widen_factor = 2, connection = (x, y) -> @. relu(x) + relu(y);
+    block_config, nclasses = 1000)
```

Comment:
Make sure to follow the style of the rest of the code base. We don't indent by only 1 on a new line. `block_config` should line up with `cardinality` here.